dor-services-client.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 2.6', '< 4'
+  spec.required_ruby_version = '>= 3.0', '< 4'
```

Comment:
hopefully this won't trip up dor-services-app

Reply:
I think it will. https://github.com/sul-dlss/dor-services-app/blob/main/Gemfile#L49https://github.com/sul-dlss/dor-services-app/blob/main/Gemfile#L49